### DISCUSSION TITLE
Looking for a reliable mapping of monitors connected through hdmi

### DISCUSSION THREAD

**Discussion by quyenvnk63** [vercel/next.js#52696] [Help]
### Summary

I'm doing a small project to manage the monitors connected to the pc through the hdmi port. I use java to display content on each screen. however I am not able to get the unique identifier of each screen so that the content can be displayed correctly for each screen. please everyone help

### Additional information

```js
i tried getting the screen's edid through EnumDisplayDevicesA but it doesn't seem to work.
```

**Comment by masterbater** [comment]
Close this way too irrelevant, probably you are from the future where shockingly nextJS became rewritten to java LOL.
